activerecord/lib/active_record/query_logs.rb

```diff
@@ -71,7 +72,10 @@ module ActiveRecord
   #
   #    config.active_record.cache_query_log_tags = true
   module QueryLogs
-    mattr_accessor :taggings, instance_accessor: false, default: {}
+    mattr_accessor :taggings, instance_accessor: false, default: {
+      line: -> { query_source_location }
```

Comment:
This should be set in `activerecord/railtie.rb` like the others, otherwise it risks being overidden by users.